tools/stern/src/Stern/Intra.hs

```diff
@@ -551,20 +551,29 @@ setTeamFeatureFlagNoConfig ::
   TeamId ->
   Public.TeamFeatureName ->
   Public.TeamFeatureStatusValue ->
+  Public.TeamFeatureTTLValue ->
   Handler ()
-setTeamFeatureFlagNoConfig tid featureName statusValue = do
+setTeamFeatureFlagNoConfig tid featureName statusValue ttlValue = do
   info $ msg "Setting team feature status for feature without config"
   gly <- view galley
   let req =
         method PUT
           . paths ["/i/teams", toByteString' tid, "features", toByteString' featureName]
           . Bilge.json (Public.TeamFeatureStatusNoConfig statusValue)
+          . Bilge.query [("ttl", convert ttlValue)]
           . contentJson
   resp <- catchRpcErrors $ rpc' "galley" gly req
   case statusCode resp of
     200 -> pure ()
     404 -> throwE (mkError status404 "bad-upstream" "team doesnt exist")
     _ -> throwE $ responseJsonUnsafe resp
+  where
+    convert = Just . toByteString' . fromDays
+    fromMinutes = (* 60)
+    fromHours = fromMinutes . (* 60)
+    fromDays = \case
+      Public.TeamFeatureTTLSeconds s -> Public.TeamFeatureTTLSeconds . fromHours . (* 24) $ s
```

Comment:
This still looks wrong.  We're parsing something that we call seconds, and then we interpret it as days.  Introduce a type `TeamFeatureTTLDays` here?  I know it's annoying, so I'll leave it to you to decide whether it's worth it.

Reply:
I went back and forth on this a few times but never got to a point I was happy with. We end up with either a "mirror" type, that does Unlimited/Days and converts to Unlimited/Seconds, or an extra, used only once constructor for Days that we can't disambiguate on parsing when receiving it via the API call.

I'm open to suggestions, but maybe this would be easier if/when we servantify stern?

Reply:
maybe the solution would be something like this:

```haskell
data TTL = TTLSecs Int | TTLUnlimited

parse "3s" = TTLSecs 3
parse "1d" = TTLSecs $ 1 * 60 * 60 * 24
...
```

anyway, yes, let's keep it as is for now.